apps/wallet-mobile/src/features/SetupWallet/common/MnemonicInput/MnemonicInput.tsx

```diff
@@ -208,95 +224,56 @@ const MnemonicWordInput = React.forwardRef<RNTextInput, MnemonicWordInputProps>(
         }
 
         if (!isEmptyString(text)) {
-          const matchingWords = getMatchingWords(text)
-          setMatchingWords(matchingWords)
+          const suggestedWords = getMatchingWords(text)
+          setSuggestedWords(suggestedWords)
 
-          if (matchingWords.length <= 0) {
+          if (suggestedWords.length <= 0) {
             setError('error')
           } else if (error !== '') setError('')
-        } else setMatchingWords([])
+        } else setSuggestedWords([])
       },
-      [error, onSubmitEditing],
+      [error, onSubmitEditing, setSuggestedWords],
     )
 
-    React.useEffect(() => {
-      if (isEmptyString(words[index])) {
-        setWord('')
-      }
-    }, [index, words])
-
     return (
-      <Menu
-        style={styles.menu}
-        contentStyle={styles.menuContent}
-        anchor={
-          <TextInput
-            ref={ref}
-            value={word}
-            onFocus={(e) => {
-              // selectTextOnFocus buggy on ios
-
-              if (Platform.OS === 'ios') {
-                e.currentTarget.setNativeProps({
-                  selection: {start: 0, end: word?.length},
-                })
-              }
-
-              onFocus()
-            }}
-            onChange={() => {
-              setMenuEnabled(true)
-              dateTime.current = Date.now()
-            }}
-            onChangeText={onChangeText}
-            enablesReturnKeyAutomatically
-            blurOnSubmit={false}
-            onSubmitEditing={onSubmitEditing}
-            dense
-            selectTextOnFocus
-            noHelper
-            errorDelay={0}
-            errorText={error}
-            autoComplete="off"
-            style={styles.textInput}
-            isPhraseValid={isPhraseValid}
-            showErrorOnBlur={false}
-            onKeyPress={({nativeEvent}) => {
-              if (nativeEvent.key === 'Backspace') {
-                onKeyPress(word)
-              }
-            }}
-            onBlur={() => {
-              setError('')
-            }}
-          />
-        }
-        visible={menuEnabled && word.length > 0 && !isEmptyString(word)}
-        onDismiss={() => {
-          setMenuEnabled(false)
-          setWord('')
+      <TextInput
+        ref={inputRef}
+        value={word}
+        onFocus={(e) => {
+          // selectTextOnFocus is buggy on ios
```

Comment:
it was fixed, we need to bump .73

Reply:
i tested and still not working on ios : |

Reply:
i will check it again when i work on the bump 👍🏻

Reply:
on .73?